src/documents/tutorial-gettingStartedWithInfusion/DefineANamespaceAndCreateAClosure.md

```diff
@@ -43,13 +43,13 @@ So what does this look like in general?
 })(jQuery, fluid_1_5);
 ```
 
-You might like to think of the [`fluid.registerNamespace`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L957-L966) call as equivalent to a line such as `var mynamespace = mynamespace || {};` written at the global scope. It is less cumbersome and more expressive of your intention, as well as easily allowing you to declare nested namespaces in one definition. Use this framework utility unless your requirements are very sophisticated (that is, you are writing a framework which you expect to be version-managed independently of both jQuery and fluid - in which case you should supply your framework's global object as an argument to the overall file closure.
+You might like to think of the [`fluid.registerNamespace`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L957-L966) call as equivalent to a line such as `var mynamespace = mynamespace || {};` written at the global scope. It is less cumbersome and more expressive of your intention, as well as easily allowing you to declare nested namespaces in one definition. Use this framework utility unless your requirements are very sophisticated (that is, you are writing a framework which you expect to be version-managed independently of both jQuery and fluid - in which case you should supply your framework's global object as an argument to the overall file closure).
```

Comment:
Regarding referencing the code: Good point. As far as I know, we don't have a strategy. We should certainly come up with one. I suspect the master docs should always reference the master code, and when we tag both, somewhere in that process we do something appropriate such that the tagged docs reference the tagged code.
For now, we should probably (since these are 'master' docs) update the links to master code, but probably not as part of this pull request.